treeherder/config/settings.py

```diff
@@ -48,6 +48,10 @@
 MEDIA_ROOT = path("media")
 MEDIA_URL = "/media/"
 
+# Default minimum regression threshold for perfherder is 1% (otherwise
+# e.g. the build size tests will alert on every commit)
+PERFHERDER_REGRESSION_THRESHOLD = 1
```

Comment:
We can make the regression threshold a per-series property. Then we can define in-tree. :)